ang/crmMailing/BlockMailing.html

```diff
@@ -6,7 +6,7 @@
 -->
 <div class="crm-block" ng-form="subform" crm-ui-id-scope>
   <div class="crm-group">
-    <div crm-ui-field="{name: 'subform.msg_template_id', title: ts('Template')}">
+    <div crm-ui-field="{name: 'subform.msg_template_id', title: ts('Template')}" ng-if="mailing.template_type='traditional'">
```

Comment:
I did a test and it still shows the template field.
Should it be `ng-if="mailing.template_type=='traditional'"` here?

Reply:
maybe @agileware-pengyi are you able to test that locally and see if it works for you?

Reply:
Fixed